fixme.cs

```diff
@@ -24,6 +24,8 @@ public ulong Calculate(int n)
             }
 
             // FIXME: Store the result in the dictionary
+            memoized.Keys = ContainsKey(n);
+            memoized.Values = Calculate(n - 1) + Calculate(n - 2);
```

Comment:
Tohle C# odmítne zkompilovat. Ale PR se ti podařilo otevřít, takže to považuji jako splněný.